src/failures/src/a320.ts

```diff
@@ -0,0 +1,9 @@
+// One can rightfully argue that this constant shouldn't be located in @flybywiresim/failures.
+// Once we create an A320 specific package, such as @flybywiresim/a320, we can move it there.
+export const A320Failure = Object.freeze({
```

Comment:
This might be pedantic but  A320Failure? Shouldn't it be prefixed as A32NX or FBW? (or just Failure)

Reply:
I always avoid prefixing variables with information that is already apparent from the context. The A320 in this variable name isn't intended as a prefix.